exporter/exporter.go

```diff
@@ -57,8 +59,106 @@ resource "lightstep_metric_dashboard" "exported_dashboard" {
 }
 `
 
-func escapeSpanQuery(input string) string {
-	return strings.Replace(input, "\"", "\\\"", -1)
+const unifiedDashboardTemplate = `
+resource "lightstep_dashboard" "exported_dashboard" {
+  project_name = var.project
+  dashboard_name = "{{.Attributes.Name}}"
+{{range .Attributes.Charts}}
+  chart {
+    name = "{{.Title}}"
+    rank = "{{.Rank}}"
+    type = "{{.ChartType}}"
+{{range .MetricQueries}}
+    query {
+      query_name          = "{{.Name}}"
+      display             = "{{.Display}}"
+      hidden              = {{.Hidden}}
+      query_string        = {{escapeQueryString .TQLQuery}}
```

Comment:
did you check that this works without the heredoc syntax? e.g. if my UQL query is just `metric cpu.utilization | delta` (no newlines), i think this will output `query_string = metric cpu.utilization | delta` -- i have no idea if terraform will be upset about the lack of quotes

Reply:
Good catch! That's definitely wrong and needs to be fixed!